mw-mobile/lib/routes/routes.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import '../screens/welcome_screen.dart';
+import '../screens/all_ways_screen.dart';
+import '../screens/personal_area_screen.dart';
+import '../screens/all_users_screen.dart';
+import '../screens/home_screen.dart';
+import '../screens/settings_screen.dart';
+import '../screens/about_project_screen.dart';
```

Comment:
Is it standard approach to use relative path in imports for dart? It's more convenient usually to use absolute path (as in typescript, java etc.) what do you think?